packages/diagram_capture/lib/diagram_capture.dart

```diff
@@ -75,47 +75,30 @@ class _Diagram extends StatelessWidget {
   }
 }
 
-const double _kDefaultDiagramViewportWidth = 1280;
-const double _kDefaultDiagramViewportHeight = 1024;
-const Size _kDefaultDiagramViewportSize = Size(
-  _kDefaultDiagramViewportWidth,
-  _kDefaultDiagramViewportHeight,
-);
-const ui.ViewConstraints _kDefaultDiagramViewportConstraints =
-    ui.ViewConstraints(
-  minWidth: _kDefaultDiagramViewportWidth,
-  minHeight: _kDefaultDiagramViewportHeight,
-  maxWidth: _kDefaultDiagramViewportWidth,
-  maxHeight: _kDefaultDiagramViewportHeight,
-);
+const Size _kDefaultDiagramViewportSize = Size(1280.0, 1024.0);
 
 // View configuration that allows diagrams to not match the physical dimensions
 // of the device. This will change the view used to display the flutter surface
 // so that the diagram surface fits on the device, but it doesn't affect the
 // captured image pixels.
 class _DiagramViewConfiguration extends ViewConfiguration {
   _DiagramViewConfiguration({
-    super.constraints = _kDefaultDiagramViewportConstraints,
+    super.size = _kDefaultDiagramViewportSize,
```

Comment:
@gspencergoog I reverted some change you made, the dynamic view size pr is reverted and this change needs to be reverted too. I think we should make sure these tests are in flutter/test

Reply:
Yeah, you're right, they should be in the customer tests.

Reply:
I will write a pr to add to flutter/test